models/calendar.yml

```diff
@@ -186,7 +186,16 @@
         title: Intermission
         emoji: 🕣
 
-    - 8:30 PM: Performer TBA
+    - 8:30 PM:
+        type: Performance
+        emoji: ⚡
+        title: Sounds All Around
+        description: I like to make interactive and immersive art.
```

Comment:
Could you wrap the description in a pair of `"`? Right now, the `'` in `we'll` is messing things up.